feedback/templates/feedback/feedback_thankyou.html

```diff
@@ -0,0 +1,15 @@
+{% extends "minimal.html" %}
+{% load i18n %}
+{% block title %}{{ block.super }} &mdash; {% trans "Thank you!" %}{% endblock %}
+{% block main %}
+            <div class="feedback-body text-center">
```

Comment:
Pretty wide indentation here, too

Reply:
Same thing.  Causes the end product to be pretty.